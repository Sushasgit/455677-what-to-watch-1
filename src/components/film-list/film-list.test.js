import React from 'react';
import renderer from 'react-test-renderer';
import FilmList from './film-list.jsx';

it(`FilmList run correctly`, () => {
  const tree = renderer
    .create(<FilmList
      data={[]}
      onClick={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
