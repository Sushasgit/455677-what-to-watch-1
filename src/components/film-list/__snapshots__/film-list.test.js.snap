// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmList run correctly 1`] = `
<div
  className="catalog__movies-list"
/>
`;
